app/uk/gov/hmrc/claimvatenrolmentfrontend/controllers/CaptureBox5FigureController.scala

```diff
@@ -64,12 +67,14 @@ class CaptureBox5FigureController @Inject()(mcc: MessagesControllerComponents,
                     matched => if (matched) {
                       Redirect(routes.CaptureLastMonthSubmittedController.show(journeyId).url)
                     } else {
+                      errorLog(s"The box 5 figure could not be updated for journey $journeyId")
                       throw new InternalServerException(s"The box 5 figure could not be updated for journey $journeyId")
                     }
               }
           )
         case None =>
-          throw new InternalServerException("Internal ID could not be retrieved from Auth")
+          errorLog(s"Internal ID could not be retrieved from Auth")
+          Future.successful(InternalServerError(errorHandler.internalServerErrorTemplate))
```

Comment:
Same here, the None on line 75 is from the authorised().retrieve(internalId) call. We would still want this to throw an error as this would mean something as gone wrong with auth.

This needs to be updated on the other controllers too